bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -150,6 +150,28 @@ public RotelHandler(Thing thing, RotelStateDescriptionOptionProvider stateDescri
     public void initialize() {
         logger.debug("Start initializing handler for thing {}", getThing().getUID());
 
+        RotelThingConfiguration config = getConfigAs(RotelThingConfiguration.class);
+
+        protocol = RotelProtocol.HEX;
+        if (config.protocol != null && !config.protocol.isEmpty()) {
+            try {
+                protocol = RotelProtocol.getFromName(config.protocol);
+            } catch (RotelException e) {
+                // Invalid protocol name in configuration, HEX will be considered by default
```

Comment:
Perhaps this should result in OFFLINE with CONFIGURATION_ERROR? The user might have tried to configure another protocol and made a typo, and now will unexpectedly and silently be using HEX. Having a clear error shown might save some time trouble-shooting this?

Reply:
Now saw that this is moved code, so resolve comment if you prefer not to improve this in context of this PR.

Reply:
I will check that later but I guess it should be already checked and handled by the core framework (thing configuration validation). So the exception could probably never happened in practice. 
I will create another PR if necessary.